test/CMakeLists.txt

```diff
@@ -16,8 +16,8 @@ if(NOT TARGET gtest)
   include(FetchContent)
   FetchContent_Declare(
     googletest
-    URL https://github.com/google/googletest/archive/609281088cfefc76f9d0ce82e1ff6c30cc3591e5.zip
-  )
+    GIT_REPOSITORY https://github.com/google/googletest.git
```

Comment:
The same content declaration which works Windows / WSL.
https://stackoverflow.com/a/24232441

Reply:
Why not to use the last release version? It looks better than "in-the-middle-of-nowhere" commit

Reply:
I wasn't sure if it also was a random commit. Created a new issue about it.